end_to_end/test_convergence_1b_params.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+USER=${1}
+LOSS_THRESHOLD=${2}
+OUTPUT_PATH=${3}
+DATASET_PATH=${4}
+
+
+if [ -z ${5} ]
+then 
+    RUN_NAME=${USER}
+else
+    RUN_NAME=${5}
+fi
+
+# Train
+export LIBTPU_INIT_ARGS="--xla_tpu_enable_data_parallel_all_reduce_opt=true --xla_tpu_data_parallel_opt_different_sized_ops=true --xla_tpu_enable_async_collective_fusion=true --xla_tpu_enable_async_collective_fusion_fuse_all_gather=true --xla_tpu_enable_async_collective_fusion_multiple_steps=true --xla_tpu_overlap_compute_collective_tc=true --xla_enable_async_all_gather=true"
+python3 MaxText/train.py MaxText/configs/base.yml run_name=$RUN_NAME\
```

Comment:
Also does this work? Isn't our loss ramp set to 15000 steps by default?
https://github.com/google/maxtext/blob/main/MaxText/configs/base.yml#L130